src/OSPSuite.UI/Views/Importer/MappingParameterEditorView.Designer.cs

```diff
@@ -142,7 +142,7 @@ private void InitializeComponent()
          // 
          this.unitsLayoutControlItem.Control = this.unitsPanelControl;
          this.unitsLayoutControlItem.Location = new System.Drawing.Point(0, 0);
-         this.unitsLayoutControlItem.Name = "Units";
+         this.unitsLayoutControlItem.Name = Captions.UnitsEditorCaption;
```

Comment:
THis should be done in InitializeResouces method in the view. We never change the Designer file by hand